bitnami/seaweedfs/Chart.yaml

```
@@ -50,4 +50,4 @@ name: seaweedfs
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/seawwedfs
 - https://github.com/bitnami/containers/tree/main/bitnami/seaweedfs
-version: 4.2.1
+version: 4.2.2
```

Let's bump a minor since a new parameter is added
```suggestion
version: 4.3.0
```

---

Sure. Thanks!